src/issuer/issuer.service.ts

```diff
@@ -61,9 +61,9 @@ export class IssuerService {
     /**
      * Record the issued credential using the external credential record service
      */
-    public async recordCredential(entityData: any, issuanceRecord: any) {
+    public async recordCredential(entityData: any, issuanceRecord: any): Promise<void> {
         const url = process.env.CREDENTIAL_RECORD_URL;
-        this.callService('POST', url, {
+        await this.callService('POST', url, {
```

Comment:
I'm not sure if it's safe to `await` here, it might be intentionally asynchronous. @matt-raffel-kiva do you know?